homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -426,7 +427,12 @@ def unique_id(self) -> str:
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
-        return {"device_id": self.device_id, "zone_id": self._device["zone"]}
+        return {"device_id": self.device_id, "zone_id": self._device.get("zone")}
```

Comment:
If the property doesn't exist we usually don't set the key instead

Reply:
ok, I've adjusted the property to exclude zone_id if it isn't present instead of returning None